spec/cocina/models/mapping/to_mods/title_spec.rb

```diff
@@ -127,7 +133,7 @@
             xsi:schemaLocation="http://www.loc.gov/mods/v3 http://www.loc.gov/standards/mods/v3/mods-3-6.xsd">
             <titleInfo type="uniform" nameTitleGroup="1">
               <title>Concertos, recorder, string orchestra</title>
-              <partNumber>RV 441, C minor</partNumber>
+              <partNumber>RV 442, F major</partNumber>
```

Comment:
Demonstrates that the part label in id metadata is used, not the one in desc metadata

Reply:
I love that you changed it to another real Vivaldi composition.

Reply:
Ha! I had to look up what the heck "RV 441" even meant. At that point, I *had* to honor precedent. 😆